extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcProviderClient.java

```diff
@@ -65,6 +67,7 @@ public OidcProviderClient(WebClient client,
         this.clientJwtKey = OidcCommonUtils.initClientJwtKey(oidcConfig);
         this.introspectionBasicAuthScheme = initIntrospectionBasicAuthScheme(oidcConfig);
         this.filters = filters;
+        this.clientSecretQueryAuthentication = oidcConfig.credentials.clientSecret.method.orElse(Method.BASIC) == Method.QUERY;
```

Comment:
Why check for Basic? A null check would make it less complicated IMHO
```suggestion
        this.clientSecretQueryAuthentication = oidcConfig.credentials.clientSecret.method.orElse(null) == Method.QUERY;
```